RobotAppEyes/RobotAppEyes.py

```diff
@@ -70,66 +102,68 @@ class RobotAppEyes:
     | NAME              | Check Eyes Region By Selector `|` NAME              `|` my_element               `|`  NameElement             | Matches by @name attribute                      |
     | TAG NAME          | Check Eyes Region By Selector `|` TAG NAME          `|` div                      `|`  TagNameElement          | Matches by HTML tag name                        |
     """
+
     ROBOT_LIBRARY_SCOPE = 'GLOBAL'
-    ROBOT_LIBRARY_VERSION = VERSION
 
     def open_eyes_session(self,
+                          applitoolsurl,
```

Comment:
It would be really nice to leave this argument as an optional argument. By making this a mandatory argument we make a breaking change in the code which would make it harder for people to upgrade to this new code